packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker-task-base-types.ts

```diff
@@ -184,20 +184,15 @@ export interface ResourceConfig {
  * @experimental
  */
 export interface VpcConfig {
-    /**
-     * VPC security groups.
-     */
-    readonly securityGroups: ec2.ISecurityGroup[];
-
     /**
      * VPC id
      */
-    readonly vpc: ec2.Vpc;
+    readonly vpc: ec2.IVpc;
 
     /**
      * VPC subnets.
      */
-    readonly subnets: ec2.ISubnet[];
+    readonly subnets?: ec2.ISubnet[];
```

Comment:
Normally we ask people to pass a subnetselection here. See other constructs that do the same.

Reply:
Have changed to pass ec2,SubnetSelection instead of ec2.ISubnet[]